.github/workflows/downstream_inclusion.yml

```
@@ -0,0 +1,153 @@
+---
+# This file is part of the NiAS project (https://github.com/nias-project).
+# Copyright NiAS developers and contributors. All rights reserved.
+# License: BSD 2-Clause License (https://opensource.org/licenses/BSD-2-Clause)
+name: downstream inclusion
```

Just to align with the other workflows^^
```suggestion
name: Downstream inclusion
```